public/app/plugins/datasource/cloudwatch/language/cloudwatch-sql/completion/CompletionItemProvider.ts

```diff
@@ -179,12 +179,15 @@ export class SQLCompletionItemProvider extends CompletionItemProvider {
               dimensionFilters = (labelKeyTokens || []).reduce((acc, curr) => {
                 return { ...acc, [curr.value]: null };
               }, {});
-              const keys = await this.resources.getDimensionKeys({
-                namespace: this.templateSrv.replace(namespaceToken.value.replace(/\"/g, '')),
-                region: this.templateSrv.replace(this.region),
-                metricName: metricNameToken?.value,
-                dimensionFilters,
-              });
+              const keys = await this.resources.getDimensionKeys(
+                {
+                  namespace: this.templateSrv.replace(namespaceToken.value.replace(/\"/g, '')),
+                  region: this.templateSrv.replace(this.region),
+                  metricName: metricNameToken?.value,
+                  dimensionFilters,
+                },
+                false
```

Comment:
This disabled the warning about multi-template variables since Metric Search queries support a comma separated list of arguments for `FROM SCHEMA` and `GROUP BY`.

> If you specify SCHEMA with additional arguments, the additional arguments after the namespace argument must be label keys. Label keys must be dimension names. If you specify one or more of these label keys, the results are scoped down to only those metrics that have that exact set of dimensions. The order of these label keys does not matter.

> Groups the query results into multiple time series, each one corresponding to a different value for the specified label key or keys. For example, using GROUP BY InstanceId returns a different time series for each value of InstanceId. Using GROUP BY ServiceName, Operation creates a different time series for each possible combination of the values of ServiceName and Operation.

From https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch-metrics-insights-querylanguage.html